homeassistant/components/workday/config_flow.py

```diff
@@ -0,0 +1,305 @@
+                CONF_COUNTRY: combined_input[CONF_COUNTRY],
+                CONF_EXCLUDES: combined_input[CONF_EXCLUDES],
+                CONF_OFFSET: combined_input[CONF_OFFSET],
+                CONF_WORKDAYS: combined_input[CONF_WORKDAYS],
+                CONF_ADD_HOLIDAYS: combined_input[CONF_ADD_HOLIDAYS],
+                CONF_REMOVE_HOLIDAYS: combined_input[CONF_REMOVE_HOLIDAYS],
+                CONF_PROVINCE: combined_input[CONF_PROVINCE],
+            }
+
+            self._async_abort_entries_match(abort_match)
+            if not errors:
+                return self.async_create_entry(
+                    title=combined_input[CONF_NAME],
+                    data={},
+                    options=combined_input,
+                )
+
+        schema = await self.hass.async_add_executor_job(
+            add_province_to_schema, DATA_SCHEMA_OPT, self.data
+        )
+        new_schema = self.add_suggested_values_to_schema(schema, user_input)
+        return self.async_show_form(
+            step_id="options",
+            data_schema=new_schema,
+            errors=errors,
+        )
+
+
+class WorkdayOptionsFlowHandler(OptionsFlowWithConfigEntry):
+    """Handle Workday options."""
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Manage Workday options."""
+        errors: dict[str, str] = {}
+
+        if user_input is not None:
+            combined_input: dict[str, Any] = {**self.options, **user_input}
+            if combined_input[CONF_PROVINCE] == "none":
+                combined_input[CONF_PROVINCE] = None
+
+            try:
+                await self.hass.async_add_executor_job(
+                    validate_custom_dates, combined_input
+                )
+            except AddDatesError:
+                errors["add_holidays"] = "add_holiday_error"
+            except RemoveDatesError:
+                errors["remove_holidays"] = "remove_holiday_error"
+            else:
+                try:
+                    self._async_abort_entries_match(
+                        {
+                            CONF_COUNTRY: self._config_entry.options[CONF_COUNTRY],
+                            CONF_EXCLUDES: combined_input[CONF_EXCLUDES],
+                            CONF_OFFSET: combined_input[CONF_OFFSET],
+                            CONF_WORKDAYS: combined_input[CONF_WORKDAYS],
+                            CONF_ADD_HOLIDAYS: combined_input[CONF_ADD_HOLIDAYS],
+                            CONF_REMOVE_HOLIDAYS: combined_input[CONF_REMOVE_HOLIDAYS],
+                            CONF_PROVINCE: combined_input[CONF_PROVINCE],
+                        }
+                    )
+                except AbortFlow as err:
+                    errors = {"base": err.reason}
+                else:
+                    return self.async_create_entry(
+                        title="",
+                        data=combined_input,
+                    )
```

Comment:
title is no longer needed for option flows.
```suggestion
                    return self.async_create_entry(data=combined_input)
```